spec/qiita/markdown/filters/code_block_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+describe Qiita::Markdown::Filters::CodeBlock do
+  subject(:filter) { described_class.new(input_html) }
+
+  let(:context) { nil }
+
+  context "without code" do
+    let(:input_html) do
+      <<~HTML
+        <pre>
+        </pre>
+      HTML
+    end
+
+    it "does not change" do
+      expect(filter.call.to_s).to eq(input_html)
+    end
+  end
+
+  context "with code" do
+    let(:input_html) do
+      <<~HTML
+        <pre><code>
+        </code></pre>
+      HTML
+    end
+
+    it "does not change" do
+      expect(filter.call.to_s).to eq(input_html)
+    end
+
+    context "with data-metadata" do
+      let(:input_html) do
+        <<~HTML
+          <pre><code data-metadata>
+          </code></pre>
+        HTML
+      end
+
+      it "does not change" do
+        expect(filter.call.to_s).to eq(input_html)
+      end
+
+      context "with data-metadata value" do
+        let(:input_html) do
+          <<~HTML
+            <pre><code data-metadata="ruby">
+            </code></pre>
+          HTML
+        end
+
+        let(:output_html) do
+          <<~HTML
+            <pre lang="ruby"><code data-metadata="ruby">
+            </code></pre>
+          HTML
+        end
+
+        it "adds lang on pre " do
```

Comment:
[nits] 文末に空白が入ってます。他の箇所にもあります。
```suggestion
        it "adds lang on pre" do
```

Reply:
Thank you for reviewing
I fixed it.